charts/hdfs-datanode-k8s/templates/datanode-daemonset.yaml

```diff
@@ -82,3 +159,17 @@ spec:
           hostPath:
             path: {{ $path }}
         {{- end }}
+        {{- if .Values.kerberosEnabled }}
+        - name: kerberos-config
+          configMap:
+            name: {{ .Values.kerberosConfigMap }}
+        - name: kerberos-keytabs
```

Comment:
Should this be renamed to `hdfs-kerberos-keytabs` as in values.yaml and line 59 of README.md?

Reply:
This is a volume name that is internal to the daemonset yaml file. So it doesn't need to match the external config map name `hdfs-kerberos-keytabs`. `{{ .Values.kerberosConfigMap }}` will match the external config map name. I was hoping to use the shorter name inside the yaml file because the long name is mouthful and this appears as a prefix in multiple places.